load-testing/avl-consumer-subscriptions.js

```diff
@@ -0,0 +1,102 @@
+import { check } from "k6";
+import exec from "k6/execution";
+import http from "k6/http";
+
+const SMALL_PRODUCER_SUBSCRIPTION_ID = "14964";
```

Comment:
It might be nice to add a README to this to explain to how to run the load test and any pre-reqs e.g. that AVL  subscriptions with ID 14964 and 3492 should exist before trying to run the tests